extensions/amp-story/1.0/page-advancement.js

```diff
@@ -475,6 +475,13 @@ export class ManualAdvancement extends AdvancementConfig {
           return true;
         }
 
+        if (
+          el.classList.contains('i-amphtml-story-screen-reader-back-button')
+        ) {
+          shouldHandleEvent = false;
```

Comment:
shouldn't this event be actually handled by `ManualAdvancement`? Confused about how this works.